cmd/kube-proxy/app/server.go

```diff
@@ -107,6 +108,25 @@ const (
 	proxyModeIptables               = "iptables"
 	experimentalProxyModeAnnotation = "net.experimental.kubernetes.io/proxy-mode"
 	betaProxyModeAnnotation         = "net.beta.kubernetes.io/proxy-mode"
+	iptables_kmod_err               = "Check if iptables has been enabled in the kernel. " +
+		"Make sure the following kernel modules (or in kernel features) are compiled and loaded:\n" +
+		"xt_conntrack\n" +
```

Comment:
Thanks for catching that. Will update this. Might have to refactor this PR based on @thockin though.